app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,120 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+You could inspire your pupils to consider rewarding careers from tackling climate change to artificial intelligence (AI).
+  </p>
+  <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee physics teachers</a>.</p>
+
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Spark curiosity in physics",
+      colour: "yellow",
+      image_1_path: "static/images/content/hero-images/0012.jpg",
+      image_1_alt: "Pupils listening to a science lesson.",
+      image_2_path: "static/images/content/hero-images/0007.jpg",
+      image_2_alt: "Teacher talking to a pupil in a classroom.",
+    ) do %>
+       <p>As a physics teacher, you’ll be engaging students in a wide range of experiments that can spark their interest.</p>
+<p>You can inspire your students to consider rewarding careers from tackling climate change to artificial intelligence.  </p>
+
+  <p>
+  </p>
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+      <p>
+      You'll teach the content of the national curriculum, with opportunities to develop your pupils' ability to think like physicists and see physics as a solution to many challenges. The themes you are likely to cover for 11 to 14 year olds (key stage 3) include:</p>
+        <ul class="vertical-tags">
+          <li class="tag">Energy</li>
+          <li class="tag">Motion and forces</li>
+          <li class="tag">Electricity and electromagnetism</li>
+          <li class="tag">Waves and matter</li>
+        </ul>
+      <p>
+        Themes you’ll cover for 14 to 16 year olds (key stage 4):
```

Comment:
@caitlin-reidy as above it would be good to include some examples of what they'll be teaching to make this more relatable. But we can revisit once we get this info.